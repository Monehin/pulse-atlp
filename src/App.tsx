import React from 'react';
import { css, jsx } from '@emotion/core';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
/** @jsx jsx */

import Home from './components/Home/Home';
import Dashboard from './components/Dashoard/Dashboard';
import UserProvider from './Providers/UserProvider';
import PrivateRoute from './components/PrivateRoute/PrivateRoute';

const appStyle = css``;

function App() {
  return (
    <UserProvider>
      <Router>
        <div className='app' css={appStyle}>
          <Switch>
            <Route exact path='/' component={Home} />
            <PrivateRoute path='/dashboard' component={Dashboard} />
          </Switch>
        </div>
      </Router>
    </UserProvider>
  );
}

export default App;
