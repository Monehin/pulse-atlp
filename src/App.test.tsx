import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

describe('Dummy Suite', () => {
  it('Dummy test', (done) => {
    expect(1 === 1).toEqual(true);
    done();
  });
});
