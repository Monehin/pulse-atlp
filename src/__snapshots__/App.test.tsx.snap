// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage render 1`] = `
<div
  className="css-osdkme-Loading"
>
  <div
    className="obj"
  />
  <div
    className="obj"
  />
  <div
    className="obj"
  />
  <div
    className="obj"
  />
  <div
    className="obj"
  />
  <div
    className="obj"
  />
  <div
    className="obj"
  />
  <div
    className="obj"
  />
  <div
    className="obj"
  />
</div>
`;
